packages/fxa-auth-server/lib/error.js

```diff
@@ -1205,7 +1205,7 @@ AppError.invalidOrExpiredOtpCode = () => {
     code: 400,
     error: 'Bad Request',
     errno: ERRNO.INVALID_EXPIRED_OTP_CODE,
-    message: 'Invalid or expired OTP code',
+    message: 'Invalid or expired verification code',
```

Comment:
This *should* have been handled in the content-server https://github.com/mozilla/fxa/blob/033507d45a9d4eea394006e2f1bc9377ae9ed004/packages/fxa-content-server/app/scripts/lib/auth-errors.js#L271, but it wasn't 😳. I'll investigate a little more why that isn't that case.

Reply:
@vbudhram The PR queue is getting a bit out of hand, so I've filed https://github.com/mozilla/fxa/issues/2701 to follow up on your question.

Reply:
Changes to the auth-server error module does not affect what is displayed in the content server. For that to happen, we have to ensure the errno returned here has an entry in [app/scripts/lib/auth-errors.js](https://github.com/mozilla/fxa/blob/483f18b15b830b11f010a89b8e88c08031f40daa/packages/fxa-content-server/app/scripts/lib/auth-errors.js#L36). This translation layer ensures error strings are translated for the user's locale.